src/app/dashboard/filter-ops/filter-ops.component.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, Input } from '@angular/core';
+
+import { filterConf } from '../../_helpers/filter-conf';
+import {
+  FilterExecution,
+  FilterExecutionOption,
+  FilterParamHash,
+  FilterParamType,
+  FilterParamValue,
+} from '../../_models/filterExecution';
+
+@Component({
+  selector: 'app-filter-ops',
+  templateUrl: './filter-ops.component.html',
+  styleUrls: ['./filter-ops.component.scss'],
+})
+export class FilterOpsComponent {
+  showing = false;
+  conf: FilterExecution[];
```

Comment:
The form is configured so that parameters can easily be added and removed.  The configuration is a series of filters for executions - hence the (array) FilterExecution.  

To make more this explicit I renamed:
  FilterExecution > FilterExecutionConf

the submodel is now clunkier, but it's explicit:

  FilterExecutionOption > FilterExecutionConfOption
  FilterExecutionOptionInput > FilterExecutionConfOptionInput

Reply:
The config file has been moved - see comment above